varnish/datadog_checks/varnish/varnish.py

```diff
@@ -293,8 +296,8 @@ def _parse_varnishadm(self, output, tags):
             backend, status, message = None, None, None
             # split string and remove all empty fields
             tokens = filter(None, line.strip().split(' '))
-
-            if len(tokens) > 0:
+            tokens = [t for t in tokens]
```

Comment:
why'd you add this line?

Reply:
The `additional notes` in the PR description explains this. In Python3 you can't take the length of a `filter` since it now returns an iterator. I did this to avoid changing the core functionality of the check as much as possible just to port to Py3

Reply:
ope I didn't see that! Thanks!

Reply:
No problem!